templates/inspec/integration_test_template.erb

```diff
@@ -0,0 +1,17 @@
+<%= compile 'templates/license.erb' -%>
+<%= lines(autogen_notice :ruby) -%>
+
+require 'vcr_config'
+
+title 'Test GCP <%= name -%> resource.'
+
+<%= compile("templates/inspec/examples/attributes/#{name}.erb") -%>
+
+control '<%= name -%>-1.0' do
+  impact 1.0
```

Comment:
This is just my curiosity, but what does `impact 1.0` mean?

Reply:
Because InSpec is built for compliance testing it has a concept of the importance of an individual test from 0-1. This doesn't really make sense for testing the resources, because any failure would be important.